image/templates/helm/stackrox-secured-cluster/internal/config-shape.yaml

```diff
@@ -57,6 +57,7 @@ image:
     name: null # string
     tag: null # string
     fullRef: null # string
+  scannerV4: null # dict
```

Comment:
just curious why is this a `dict` vs specifying the fields like we do for the others?

Reply:
oversight. Thanks.